styles/styles.css

```
@@ -168,6 +169,22 @@ strong {
   font-family: var(--ff-proxima-xbold);
 }
 
+em {
+  font-style: italic;
+}
+
+em strong {
+  font-family: var(--ff-proxima-regular);
+  font-weight: 700;
+}
+
+.text-caption {
```

Please prefix with `main`
```suggestion
main .text-caption {
```